src/bracuganda/res/values/strings.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 	<string name="app_name">BRAC</string>
-	<string name="fixed_app_url">https://brac-ug.app.medicmobile.org</string>
+	<string name="scheme">https</string>
```

Comment:
We only support `https` so this value should go in the `src/main/res/values/strings.xml` file I think...

Reply:
Done! I didn't even see that file before :D